Multisig/UI/Settings/OwnerKeyManagement/OnboardingImportOwnerKeyViewController.swift

```diff
@@ -13,7 +13,7 @@ class OnboardingImportOwnerKeyViewController: UIViewController {
     @IBOutlet weak var tableView: UITableView!
     private var nextButton: UIBarButtonItem!
 
-    private let cards = [
+    private let cards: [(image: UIImage?, title: String, body: String, link: (title: String?, url: URL?))] = [
```

Comment:
My bad not pointing out 😄  I was hoping you'll use this named tuple on lines 64-67

Reply:
Maybe it's time to create structure instead of tuple, I believe it might be used in future in other onboarding views

Reply:
Let's refactor it in future if needed?